index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style/tailwind.css" />
+    <link rel="stylesheet" href="./dist/tailwind.css">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <title>Google</title>
+  </head>
+  <body class="relative flex flex-col justify-between h-screen font-arial">
+    <section class="relative top-0 grid w-full md:fixed">
+      <ul class="flex items-center justify-end gap-4 p-6">
+        <li class=" hover:underline text-primarySize"><a href="#">Gmail</a></li>
+        <li class=" hover:underline text-primarySize"><a href="#">Images</a></li>
+        <img
+          src="./asserts/apps_FILL0_wght500_GRAD0_opsz24.svg"
+          alt="application"
+          class="px-2 pt-2 pb-2 mt-auto mb-auto rounded-full cursor-pointer hover:bg-primary-color"
+        />
+        <img
+          src="./asserts/profil.jpg"
+          alt="user-profil"
+          class="border-2 rounded-full hover:border-gray-400 h-9"
+        />
+      </ul>
+    </section>
+    <section class="flex flex-col items-center justify-center m-auto">
+      <img
+        src="./asserts/googlelogo_color_272x92dp.png"
+        alt="google-logo"
+        class="mb-8 md:w-80 lg:w-52"
+      />
+      <form action="" class="flex justify-center hover:drop-shadow-md">
+        <div class="relative basis-[35rem] flex items-center flex-grow flex-shrink-0 ">
+          <img src="./asserts/rech-removebg-preview.png" alt="serch-icon" class="absolute h-12 ml-3 text-secondary-color material-symbols-outlined">
+          <input
+            type="text"
+            class="w-full p-2 px-40 pl-10 pr-10 border-none rounded-full outline-none ring-1 ring-gray-200"
+          />
+          <img
+            src="./asserts/cam-removebg-preview.png"
+            class="absolute cursor-pointer h-9 right-2"
+            alt="logo"
+          />
+        </div>
+      </form>
+      <section class="flex flex-col items-center justify-center gap-1 p-6">
+        <div class="flex space-x-4 text-secondarySize">
+          <button
```

Comment:
add the hover effects and match the background with the exact google background. you may also add some holizontal padding